x-pack/filebeat/module/suricata/eve/ingest/pipeline.json

```diff
@@ -19,83 +19,93 @@
       }
     }
   , { "set":
-      { "field": "ecs.event.type"
+      { "field": "event.type"
       , "value": "{{suricata.eve.event_type}}"
       }
     }
 
   , { "set":
-      { "field": "ecs.user_agent.raw"
+      { "field": "user_agent.original"
       , "value": "{{suricata.eve.http.http_user_agent}}"
       }
     }
   , { "user_agent":
-      { "field": "ecs.user_agent.raw"
-      , "target_field": "ecs.user_agent"
+      { "field": "user_agent.original"
+      , "target_field": "user_agent"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os"
-      , "target_field": "ecs.user_agent.temp_os"
+      { "field": "user_agent.os"
+      , "target_field": "user_agent.temp_os"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.temp_os"
-      , "target_field": "ecs.user_agent.os.full_name"
+      { "field": "user_agent.temp_os"
+      , "target_field": "user_agent.os.full_name"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_name"
-      , "target_field": "ecs.user_agent.os.name"
+      { "field": "user_agent.os_name"
+      , "target_field": "user_agent.os.name"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_version"
-      , "target_field": "ecs.user_agent.os.version"
+      { "field": "user_agent.os_version"
+      , "target_field": "user_agent.os.version"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_major"
-      , "target_field": "ecs.user_agent.os.major"
+      { "field": "user_agent.os_major"
+      , "target_field": "user_agent.os.major"
       , "ignore_missing": true
       }
     }
   , { "rename":
-      { "field": "ecs.user_agent.os_minor"
-      , "target_field": "ecs.user_agent.os.minor"
+      { "field": "user_agent.os_minor"
+      , "target_field": "user_agent.os.minor"
       , "ignore_missing": true
       }
     }
 
   , { "geoip":
       { "field": "suricata.eve.src_ip"
-      , "target_field": "ecs.source.geo"
+      , "target_field": "suricata.eve.src_geo"
```

Comment:
ECS doesn't yet have a `source.geo` or a `destination.geo` but [it will](https://github.com/elastic/ecs/issues/94). So for master, where we don't have the `source` field conflict, we should put the geo data under those fields.

Reply:
Do you already have a plan for dealing with the ECS `source` conflict with the Filebeat `source` for 6.x?

Perhaps we could just rename all uses of the ECS `source` field to `source_ecs` when we backport.

Reply:
My idea was to have this branch directly mergeable into 6.x to speed up things, so no `source` usage yet.

I thought it makes more sense to not create an intermediate field (i.e. `source_ecs`) just to have it replaced with `source` soon. It made more sense to leave this under `suricata.eve` for now and keep those fields too for 7.x, adding also `source`.

Makes any sense?